cloudformation/stacks/AutoSpotting/parameters.yaml

```diff
@@ -1,3 +1,8 @@
 ---
 LambdaHandlerFunction: handler.handle
 LambdaS3Bucket: cloudprowess
+LambdaMemorySize: '256'
+ExecutionFrequency: "rate(1 minute)"
+MinOnDemandNumber: '1'
+MinOnDemandPercentage: '50'
```

Comment:
This is just the way I execute my testing stack, it's merely an example. But you're right, I will change it to actually make some sense.